Bloom/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Fluent",
+    "description": "Spicetify theme inspired by Microsoft's Fluent Design",
+    "preview": "https://raw.githubusercontent.com/williamckha/spicetify-fluent/master/screenshots/dark-1.png",
+    "readme": "README.md",
+    "usercss": "user.css",
+    "schemes": "color.ini",
+    "include": ["https://raw.githubusercontent.com/williamckha/spicetify-fluent/master/fluent.js"],
+    "branch": "master",
+    "authors": [
+        {
+            "name": "William Ha",
+            "url": "https://github.com/williamckha"
+        }
+    ]
+}
```

Comment:
Okay, so this file needs to be deleted and its contents moved to the root manifest, since this is the file that allows for the download from Marketplace. Also, update the names and file names present so it reflects "bloom" and not the OG Fluent.